app/src/main/java/mozilla/lockbox/view/ItemDetailFragment.kt

```diff
@@ -102,6 +90,20 @@ class ItemDetailFragment : BackableFragment(), ItemDetailView {
         inputLayoutPassword.isHintAnimationEnabled = false
 
         inputUsername.readOnly = true
+        when (showUsernamePlaceholder) {
+            true -> {
+                view!!.btnUsernameCopy.setColorFilter(resources.getColor(R.color.white_60_percent))
+                view!!.isClickable = false
+                view!!.isFocusable = false
+                inputUsername.setText(" ", TextView.BufferType.NORMAL)
+            }
+            false -> {
+                inputUsername.isClickable = true
+                inputUsername.isFocusable = true
+                inputUsername.setText(item.username, TextView.BufferType.NORMAL)
+            }
```

Comment:
This `when` statement is difficult to read. Perhaps it could do with a comment or two. 

1. On one hand you're dealing with the view, on the other, you're dealing with the `inputUsername`.
2. `when` on booleans still look weird to me. This may be a stylistic things.

Reply:
I could change it to an if/else. Might be easier to read. I am not changing the original code, just grouping it based on the username being available.